container-search/src/test/java/com/yahoo/prelude/test/IntegrationTestCase.java

```diff
@@ -1,174 +0,0 @@
-// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
-package com.yahoo.prelude.test;
-
-import com.yahoo.search.result.Hit;
-import com.yahoo.search.Result;
-import com.yahoo.search.Searcher;
-import com.yahoo.search.searchchain.Execution;
-import org.junit.jupiter.api.Test;
-
-/**
- * Runs a query thru the configured search chain from a real http channel
- * to a mock fdispatch channel. The setup is rather complicated, as the point is
- * to span as much of the processing from query to result as possible.
- *
- * @author bratseth
- */
-public class IntegrationTestCase {
```

Comment:
Were the changes in this files intentional?

Reply:
Intention was to remove test as everything was already commented out, and the comments and the code indicated it had been dead before it was born.